sw/device/lib/runtime/hart.c

```diff
@@ -20,12 +20,12 @@ void usleep(uint32_t usec) {
 }
 
 noreturn void abort(void) {
-  if (kDeviceStopAddress != 0) {
-    mmio_region_t end_sim_addr = mmio_region_from_addr(kDeviceStopAddress);
-    // Write all ones to the address, to signal that it was `abort`. The
-    // simulator doesn't do anything with the info, but it's good to
-    // differentiate from likely `main` return codes (0, 1).
-    mmio_region_write32(end_sim_addr, 0x0, 0xFFFFFFFF);
+  if (kDeviceTestStatusAddress != 0) {
```

Comment:
This is not true. 
https://github.com/lowRISC/opentitan/blob/54aadb2ae42d1fc5763d217fa9cd404da11211b0/sw/device/lib/testing/test_status.c#L22

We want the SW to not execute *any* piece of code after asserting pass or fail. If you want to fail the test, `abort()` must not be called. Please invoke `test_status_set(kTestStatusFailed);` instead, which will write the fail flag and then abort. 

`abort()` must not do anything other than WFI forever.

Reply:
Ok, done, abort just spins forever, with wfi.